source/query-with-natural-language/prompt-natural-language-query.txt

```diff
@@ -9,6 +9,99 @@ Prompt your First Natural Language Query
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 2
+   :depth: 1
 
-Prompt your First Natural Language Query
\ No newline at end of file
+You can use MongoDB Compass to generate filter documents using the
+natural language querying feature. This feature automatically generates 
+filter documents based on prompts you provide to the MongoDB AI engine. 
+Natural language querying serves as a helpful starting point and assists
+you in writing MongoDB queries.
+
+About this Task
+---------------
+
+.. include:: /includes/fact-natural-language-query.rst
+
+The MongoDB AI engine can return both filter documents and 
+aggregations. If your prompt results in an aggregation, a popup
+appears directing you to the :guilabel:`Aggregations` tab.
+
+This page uses the :ref:`sample_mflix.movies <sample-mflix>` 
+collection from the Atlas sample dataset.
+
+Before you Begin
+----------------
+
+You must :ref:`enable the natural language querying feature<enable-natural-language-querying>`.
+
+Steps
+-----
+
+.. procedure::
+   :style: connected
+
+   .. step:: Select the :guilabel:`Documents` tab
```

Comment:
(step title -- goal)
For steps 1 and 2, what you're trying to do is navigate to the natural natural language query bar. So consider combining the two into one step with two sub-steps. Ex:

1. Navigate to the Natural Language Query Bar 
   a. Select Documents.
   b. Click Ask AI. 
   The Natural Language Query Bar displays. 

(The appears vs. displays edit is totally subjective. "Appears" is my little pet peeve because I think it makes the display sound like it happened magically. Totally fine to use whichever you want, I believe.)

Reply:
I will update appears -> displays, also updated the step and included the nested commands.